app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/AMLSController.scala

```diff
@@ -250,15 +256,19 @@ class AMLSController @Inject()(
                 val supervisoryBodyData =
                   amlsBodies.getOrElse(validForm.amlsCode, throw new Exception("Invalid AMLS code"))
 
+                val continue =
```

Comment:
This expression looks the same as the one above - can we refactor to a common method?